src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -3046,6 +3048,85 @@ namespace ngraph
                 }
             }
 
+            template <>
+            void CPU_Emitter::EMITTER_DECL(ngraph::op::QuantizedMaxPool)
+            {
+                auto qmax_pool = static_cast<const ngraph::op::QuantizedMaxPool*>(node);
```

Comment:
Instead of duplicating this code between builder and codegen, please create a builder method in mkldnn_emitter